clang/unittests/Format/FormatTestTableGen.cpp

```diff
@@ -290,6 +290,16 @@ TEST_F(FormatTestTableGen, MultiClass) {
                "}\n");
 }
 
+TEST_F(FormatTestTableGen, MultiClassesWithPasteOperator) {
+  // This is a sensitive example for the handling of the paste operators in
+  // brace type calculation.
+  verifyFormat("multiclass MultiClass1<int i> {\n"
+               "  def : Def#x<i>;\n"
+               "  def : Def#y<i>;\n"
+               "}\n"
+               "multiclass MultiClass2<int i> { def : Def#x<i>; }\n");
```

Comment:
Do you need the trailing newlines? There are dozens of them in this file. Please remove them if they are not needed.

Reply:
I come to notice that they are not required. I will make a patch for whole this file later.